deploy/keycloak/install.sh

```
@@ -8,6 +8,7 @@ if [ $# -ge 1 ]; then
 fi
 NS=keycloak
 SERVICE_NAME=keycloak
+ISTIO_ADDONS_CHART_VERSION=0.0.1-develop
```

Shellcheck (0.11.0)

[warning] ISTIO_ADDONS_CHART_VERSION appears unused. Verify use (or export if used externally).

(SC2034)